salestax.rb

```diff
@@ -0,0 +1,52 @@
+input = []
+ans = 'y'
+non_taxable = ["book", "chocolate","chocolates", "pills"]
```

Comment:
Use `%w` syntax for array of strings
https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#collections